### PR TITLE
feat: add item zip error translations

### LINKED ISSUE #30: Add item zip plugin translations

### PR DESCRIPTION
**Pull request by pyphilia**
close #30

### CODE REVIEW

File: src/langs/en/messages.json [hunk 1]
```diff
@@ -71,5 +71,9 @@
   "IMPORT_ZIP": "The ZIP archive was successfully imported",
   "IMPORT_H5P": "The H5P package was successfully imported",
   "DELETE_MEMBER": "The account was successfully deleted",
-  "UPDATE_PASSWORD": "The password was updated successfully"
+  "UPDATE_PASSWORD": "The password was updated successfully",
+
+  "INVALID_ARCHIVE_FILE": "File is not a zip archive",
+  "INVALID_FILE_ITEM": "File's properties are invalid",
+  "UNEXPECTED_EXPORT_ERROR": "Error during exporting zip"
```
Comment (spaenleh) [inline src/langs/en/messages.json:78]:
```suggestion
  "UNEXPECTED_EXPORT_ERROR": "Error while exporting zip"
```

**Review by spaenleh — APPROVED** [review]
🐯 Looks good, just a suggestion, feel free to merge 🦞

File: src/langs/en/messages.json [hunk 2]
```diff
@@ -71,5 +71,26 @@
   "IMPORT_ZIP": "The ZIP archive was successfully imported",
   "IMPORT_H5P": "The H5P package was successfully imported",
   "DELETE_MEMBER": "The account was successfully deleted",
-  "UPDATE_PASSWORD": "The password was updated successfully"
+  "UPDATE_PASSWORD": "The password was updated successfully",
+
+  "INVALID_ARCHIVE_FILE": "the submitted file is not a zip archive",
```
Comment (dialexo) [inline src/langs/en/messages.json:76]: No uppercase?
```suggestion
  "INVALID_ARCHIVE_FILE": "The submitted file is not a zip archive",
```

File: src/langs/en/messages.json [hunk 3]
```diff
@@ -71,5 +71,26 @@
...
+  "INVALID_FILE_ITEM": "The file's properties are invalid",
+  "UNEXPECTED_EXPORT_ERROR": "An error occured while exporting the zip file",
+
+  "NOT_APP_ITEM": "The targeted item is not an application item",
+  "INVALID_APP_ORIGIN": "The application does not match its registered origin is invalid",
```
Comment (dialexo) [inline src/langs/en/messages.json:81]: I think this is redundant
```suggestion
  "INVALID_APP_ORIGIN": "The application does not match its registered origin",
```

File: src/langs/en/messages.json [hunk 4]
```diff
@@ -71,5 +71,26 @@
...
+  "TOKEN_ITEM_ID_MISMATCH": "The auth token does not match the targeted item",
+  "APP_DATA_NOT_FOUND": "No app data were not found",
+  "APP_DATA_NOT_ACCESSIBLE": "The member cannot request this app data",
+  "APP_ACTION_NOT_ACCESSIBLE": "The member cannot request this app action",
+  "APP_SETTING_NOT_FOUND": "No app setting was not found",
```
Comment (dialexo) [inline src/langs/en/messages.json:86]: There's another double negation here
```suggestion
  "APP_SETTING_NOT_FOUND": "No app setting was found",
```

File: src/langs/en/messages.json [hunk 5]
```diff
@@ -71,5 +71,26 @@
...
+  "UPDATE_PASSWORD": "The password was updated successfully",
+
+  "INVALID_ARCHIVE_FILE": "the submitted file is not a zip archive",
+  "INVALID_FILE_ITEM": "The file's properties are invalid",
+  "UNEXPECTED_EXPORT_ERROR": "An error occured while exporting the zip file",
+
+  "NOT_APP_ITEM": "The targeted item is not an application item",
+  "INVALID_APP_ORIGIN": "The application does not match its registered origin is invalid",
+  "TOKEN_ITEM_ID_MISMATCH": "The auth token does not match the targeted item",
+  "APP_DATA_NOT_FOUND": "No app data were not found",
```
Comment (dialexo) [inline src/langs/en/messages.json:83]: There's adouble negation here
```suggestion
  "APP_DATA_NOT_FOUND": "No app data were found",
```

**Review by dialexo — CHANGES_REQUESTED** [review]

**Pull request merged**